src/test/resources/Logout.feature

```diff
@@ -0,0 +1,22 @@
+Feature: User can logout
+  As a user
+  I want to logout
+  So I can see login screen
```

Comment:
đây là mô tả chức năng làm gì, c có thể viết như này
 As a user
  I want to logout
  So I can logout the app

Reply:
👍